hsv_lower_bound: [41, 135, 168]
hsv_upper_bound: [88, 255, 255]
marker_colors:
  target_1: [0.0, 1.0, 0.0, 1.0]
  target_2: [0.0, 0.5, 0.5, 1.0]
z_depth_estimations:
  target_1: 0.002
  target_2: 0.002

text_marker_size: 0.05
min_contour_area: 25
max_contour_area: 1000
